mycli/clibuffer.py

```diff
@@ -1,7 +1,6 @@
 from prompt_toolkit.enums import DEFAULT_BUFFER
 from prompt_toolkit.filters import Condition
 from prompt_toolkit.application import get_app
-from .packages.parseutils import is_open_quote
```

Comment:
Here's how it is used in pgcli: https://github.com/dbcli/pgcli/blob/master/pgcli/pgbuffer.py#L15

Should we be copying that behavior instead?

Reply:
Yeah, I've looked at that code in pgcli, but its purpose seems to be to facilitate writing stored procedures with semicolons inside them. I don't think that's customary in MySQL, the idiom seems to be 
```
\DELIMITER $$$ 
...
<SQL code with ; replaced by $$$>
...
\DELIMITER ;
```